spatialmath/capsule_test.go

```diff
@@ -37,3 +38,35 @@ func TestBoxCapsuleCollision(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, dist, test.ShouldAlmostEqual, -29.69, 1e-3)
 }
+
+func TestPointCapsuleCollision(t *testing.T) {
+	o := &R4AA{
+		Theta: math.Pi / 2,
+		RX:    1,
+		RY:    0,
+		RZ:    0,
+	}
+	o2 := &R4AA{
+		Theta: math.Pi / 4,
+		RX:    0,
+		RY:    0,
+		RZ:    1,
+	}
+	pt := r3.Vector{X: 2, Y: 2, Z: 0}
+
+	c, err := NewCapsule(NewZeroPose(), 1, 10, "")
```

Comment:
This instead of calling this this should call the new function you have in `kinematics.go` to instantiate this capsule.  This would than need to move into that package